README.md

```diff
@@ -119,7 +121,40 @@ params.SplitCount = 2
 // MinSplitSize default value: 5120
 params.MinSplitSize = 7168
 
-rtManager.DownloadFiles(params)
+totalDownloaded, totalExpected, err := rtManager.DownloadFiles(params)
+```
+
+##### Downloading Files with Results Reader:
+Using `DownloadFilesWithResultReader()` service, we can download files and get the general statistics of the action (The actual number of files downloaded, and the number of files we expected to download), and the error value if it occurred.
+In addition, this service provides us with a reader through which we can iterate the downloaded files information
```

Comment:
I suggest changing the description to:
Similar to <link to download files section>, but returns a reader, which allows iterating over the details of the downloaded files. Only files which were successfully downloaded are available by the reader.